ArduPlane/mode_avoidADSB.cpp

```diff
@@ -11,3 +11,8 @@ void ModeAvoidADSB::update()
     plane.mode_guided.update();
 }
 
+void ModeAvoidADSB::_navigate()
+{
+    plane.mode_loiter.navigate();
```

Comment:
I'm not sure these things should be calling into mode_loiter to do stuff.  They *could* call update_loiter themselves.

On the Copter side we've often factored out the functionality - `land_run_vertical` / `land_run_horizontal`, for example.  That's not universal - for example, mode-auto uses mode-rtl for RTL'ing and mode-guided for guided'ing....

It's a minor point and doesn't need to be addressed, really.